src/toolchain-conventions.adoc

```diff
@@ -425,6 +425,7 @@ tag to provide extra relocations for a given vendor.
 |===
 |*Vendor*              |*Symbol*
 |Andes                 | ANDES
+|Cheriot               | XCHERIOT
```

Comment:
The X is a bit superfluous here given this is specifically for vendor relocations, and the X in extension names is just a prefix to state that what follows is a vendor extension.

Reply:
(See how none of the others include it)

Reply:
Done